### PR TITLE
PMM-7 fixed upgrade last check Date format

### PR DESCRIPTION
**Pull request by vyarosh**

### CODE REVIEW

**Comment by vyarosh** [comment]
https://pmm.cd.percona.com/blue/organizations/jenkins/pmm2-upgrade-tests/detail/pmm2-upgrade-tests/25564/pipeline

File: tests/upgradePMM_test.js [hunk 1]
```diff
@@ -490,7 +490,7 @@ Scenario(
 );
 
 if (versionMinor >= 32) {
-  Scenario(
```
Comment (peterSirotnak) [inline tests/upgradePMM_test.js:493]: Why so many skipped tests?

File: tests/upgradePMM_test.js [hunk 2]
```diff
@@ -787,7 +787,7 @@ Scenario(
   },
 );
 
-Scenario(
+Scenario.skip(
```
Comment (peterSirotnak) [inline tests/upgradePMM_test.js:790]: Test case for changes in this PR was skipped.

File: tests/upgradePMM_test.js (hunk 1 above)
Comment (vyarosh) [reply tests/upgradePMM_test.js:493]: Hmm...looks like I picked up wrong branch. I thought these changes came from merging main

**Comment by vyarosh** [comment]
@peterSirotnak It's your branch. I've my reverted changes. Sorry.

**Pull request closed without merging**